Makefile

```diff
@@ -61,21 +61,21 @@ ${BENCH_DATA}/sgemm-blaze-static.json: $(BENCH_BLAZE)
 	$(BENCH_BLAZE) --benchmark_filter="BM_gemm_static<float>*" $(BENCHMARK_OPTIONS) \
 		--benchmark_out=${BENCH_DATA}/sgemm-blaze-static.json
 
-${BENCH_DATA}/dgemm-blast-static-panel.json: $(BENCH_BLAST)
+${BENCH_DATA}/$(shell git rev-parse --short HEAD)/dgemm-blast-static-panel.json: $(BENCH_BLAST)
```

Comment:
Maybe add a variable for `$(shell git rev-parse --short HEAD)` s.t. the code is not repeated and there is a single place of change if we need to change the command?

Reply:
Done